policyengine_us/parameters/gov/states/ct/tax:income/main/unmarried.yaml

```
@@ -0,0 +1,41 @@
+description: CT unmarried individual income tax rate
```

Please rewrite other descriptions similarly, and rename this file to be single.yaml. We also need a widow.yaml (if unspecified it can match joint.yaml)

```suggestion
description: Connecticut taxes income at these rates for single filers.
```